config/internal/mariadb/mariadb-sa.yaml.tmpl

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: mariadb-sa-{{.Name}}
```

Comment:
I think it might be better if we have a `ds-pipelines-` prefix here to avoid confusion about what the SA is intended for

Reply:
Done, updated to have the ds-pipelines- prefix.